### PR TITLE
time in python

### PR DESCRIPTION
**Pull request by Sergiy-1998**
Write a Python program to get the current time in Python.

### CODE REVIEW

**Comment by python-teacher** [comment]
Please change the name of this pull request.
It should be self-descriptive. It should be clear from the name what is this pull request about.

File: hm_4.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"""
+Write a Python program to get the current time in Python.
+"""
+
+import time
+
+
+def return_time():
+    return time.ctime()
```
Comment (python-teacher) [inline hm_4.py]: Do you know other alternatives with `datetime` module?

Comment (Sergiy-1998) [reply hm_4.py]: I know

Comment (python-teacher) [reply hm_4.py]: Which one?

Comment (Sergiy-1998) [reply hm_4.py]: datetime.now()  and several others

Comment (UkrainianEngineer) [reply hm_4.py]: So, why do you decide to use `time.ctime()`?

Comment (Sergiy-1998) [reply hm_4.py]: because time.ctime () shows the same result as datetime.now ()

Comment (python-teacher) [reply hm_4.py]: But it's not popular solution.

Comment (Sergiy-1998) [reply hm_4.py]: I understood, changed

**Comment by python-teacher** [comment]
👍

**Pull request merged**